### PR TITLE
[TW-1295] Update workspace template images and instructions

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* For Partner workspaces, added a step about selecting a workspace template during workspace creation. Also updated new and existing screenshots.
* Updated other existing screenshots because the wording changed in the UI.

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md [hunk 1]
```diff
@@ -47,15 +47,21 @@ Partner Workspaces enable you to collaborate with external partners directly in
...
+    <img alt="Create Partner Workspace" src="https://assets.postman.com/postman-docs/v10/create-partner-workspace-v10-2.jpg" width="400px"/>
 
 Anyone with permissions to create workspaces can request to create a Partner Workspace, but it must be approved by a [Partner Manager](#managing-users-and-roles). If you're not a [Super Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) or Partner Manager, you must submit your request for approval. You can also add an optional note to your request. The workspace's visibility will be set to [private](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#changing-workspace-visibility) until your request is approved. Postman will notify you when a Partner Manager reviews your request and approves or denies it.
 
-<img alt="Create Partner Workspace approval" src="https://assets.postman.com/postman-docs/v10/partner-workspaces-visibility-request-v10.jpg" width="450px" />
+<img alt="Create Partner Workspace approval" src="https://assets.postman.com/postman-docs/v10/partner-workspaces-visibility-request-v10-2.jpg" width="400px" />
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:64]: Maybe consider removing this screenshot. The paragraph above explains things clearly, and this screenshot actually looks like some kind of note in documentation text--it confused me for a sec 😄

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/using-workspaces/partner-workspaces.md:64]: Yeah absolutely! I'm fine with removing it.

**Review by ramji367 — APPROVED** [review]
Lgtm!

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**